release-notes/release-notes59.md

```diff
@@ -192,13 +192,34 @@ For information on:
 The following features have been deprecated with the release of {{site.productname}} 5.9:
 
 - [The `spellchecker_select_languages` option](#thespellchecker_select_languagesoption).
+- [The BBCode (`bbcode`) plugin](#thebbcodebbcodeplugin)
+- [The Full Page (`fullpage`) plugin](#thefullpagefullpageplugin)
+- [The Legacy Output (`legacyoutput`) plugin](#thelegacyoutputlegacyoutputplugin)
 
 ### The `spellchecker_select_languages` option
 
 With the release of {{site.productname}} 5.9, the `spellchecker_select_languages` option has been deprecated and has been replaced by the `content_langs` option.
 
 For information on the `content_langs` option, see: [Spell Checker Pro - `content_langs`]({{ site.baseurl }}/plugins/premium/tinymcespellchecker/#content_langs).
 
+### The BBCode (`bbcode`) plugin
+
+The BBCode plugin (`bbcode`) has been deprecated and will be removed in the 6.0 release of {{site.productname}}.
+
+To develop and maintain a new BBCode plugin based on the {{site.productname}} BBCode plugin, you can create a fork using the [BBCode plugin source code in the TinyMCE monorepo](https://github.com/tinymce/tinymce/tree/5.8.2/modules/tinymce/src/plugins/bbcode).
```

Comment:
Should this be using the `5.9.0` tag instead of the `5.8.2` tag? Also forking from this could be problematic due to the bundling system we have setup. Should we instead point to the `tinymce-dist` repo?

Reply:
5.8.2. either is fine but eliminates the need to remove this: https://github.com/tinymce/tinymce/pull/7040/files which we're adding in 5.9.

Re: Repo, I'll get back to you on this....

Reply:
Fixed by [`b68a3bf` (#2037)](https://github.com/tinymce/tinymce-docs/pull/2037/commits/b68a3bf048828838040a517e2bc6f8bb1dd67a80)